variables.tf

```diff
@@ -64,42 +64,42 @@ variable "cbr_rules" {
     }))) }))
     enforcement_mode = string
   }))
-  description = "(Optional, list) List of CBR rules to create"
+  description = "(Optional, list) The list of context-based restrictions (CBR) rules to create."
   default     = []
 }
 
 variable "skip_iam_authorization_policy" {
   type        = bool
-  description = "Set to true to skip the creation of an IAM authorization policy that permits all Event Notification instances in the resource group to read the encryption key from the KMS instance. No policy is created if var.kms_encryption_enabled is set to false."
+  description = "Set this variable value to `true` to skip the creation of an IAM authorization policy that permits all Event Notifications instances in the resource group to read the encryption key from the key management service (KMS) instance. No policy is created if `var.kms_encryption_enabled` variable is set to `false`."
   default     = false
 }
 
 variable "kms_encryption_enabled" {
   type        = bool
-  description = "Set this to true to control the encryption keys used to encrypt the data that you store in Event Notification. If set to false, the data is encrypted by using randomly generated keys. For more info on Managing Encryption, see https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-managing-encryption"
+  description = "Set this variable value to `true` to control the encryption keys that are used to encrypt the data that you store in the Event Notifications instance. If this value is set to `false`, the data is encrypted by using randomly generated keys. For more information, see Managing encryption https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-managing-encryption."
```

Comment:
The `Managing encryption` part should be the actual link text in this line.

Reply:
alternatively, 

- Set to `true`... 
- If set to `false`,...
- see [Managing encryption](https://cloud.ibm.com/docs/event-notifications?topic=event-notifications-en-managing-encryption) (as Michelle points out. MD links are rendered in the catalog)